keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/DoubleVertex.java

```diff
@@ -75,6 +76,17 @@ public DoubleVertex matrixInverse() {
         return new MatrixInverseVertex(this);
     }
 
+    /**
```

Comment:
The method name here should be enough to describe what's going on. I'd just leave the javadoc for the class itself.